README.md

```diff
@@ -24,37 +24,42 @@
 - 🚀 [TestFlight preview for iOS](https://testflight.apple.com/join/DWuT2LQu) with the new features 1-2 weeks earlier - **limited to 10,000 users**!
 - 🖥 Available as a terminal (console) app / CLI on Linux, MacOS, Windows.
 
-## SimpleX unique approach to privacy and security
+## Why privacy of communications matter
 
-Everyone should care about privacy and security of their communications - even ordinary conversations can put you in danger.
+Everyone should care about privacy and security of their communications - ordinary conversations can put you in danger even if you are innocent.
```

Comment:
I think it would be better to say something like "ordinary innocuous conversations can put you in danger" or "even if you have nothing to hide"

"even if you are innocent" somehow to me feels like you're pitching to the guilty and the innocents are a secondary user type